README.md

```diff
@@ -145,9 +145,13 @@ things that authors can do to help keep the conversation moving along.
 
 The following labels may be applied to enhancements to help categorize them:
 
-- `priority/important-soon` indicates that the enhancement is related to a
-top level release priority. These will be highlighted in the
-[this-week](this-week/) newsletters.
+- `priority/high`  indicates that the enhancement is related to a
+top level release priority. These will be highlighted in the [this-week](this-week/)
+newsletters.
+- `priority/medium`  indicates that the enhancement is related to a
+future release but it is not the next OCP release 
```

Comment:
There is a gap between high and medium - what priority has a feature that targets the next release, but is not as a "top level release priority"/

Reply:
yes, I have added `important-soon` and changed the wording for high priority as the discussion.